pkg/workers/notifications/notification.go

```diff
@@ -0,0 +1,211 @@
+package notifications
+
+import (
+	"crypto/ecdsa"
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/cozy/cozy-stack/pkg/config"
+	"github.com/cozy/cozy-stack/pkg/jobs"
+
+	multierror "github.com/hashicorp/go-multierror"
+
+	fcm "github.com/appleboy/go-fcm"
+	apns "github.com/sideshow/apns2"
+	apns_cert "github.com/sideshow/apns2/certificate"
+	"github.com/sideshow/apns2/payload"
+	apns_token "github.com/sideshow/apns2/token"
+)
+
+const (
+	// Android platform using Firebase Cloud Messaging (FCM)
+	Android = "android"
+	// IOS platform using APNS/2
+	IOS = "ios"
+)
+
+var (
+	fcmClient *fcm.Client
+	iosClient *apns.Client
+)
+
+func init() {
+	jobs.AddWorker("notification", &jobs.WorkerConfig{
```

Comment:
I'd have named this package and this woker `push`, not `notification`. Why this choice?

Reply:
Indeed, I like `push` better too. It also avoid confusions with our `pkg/notifications` package.